package.json

```diff
@@ -41,7 +41,7 @@
     "preq": "^0.4.4",
     "restbase-mod-table-cassandra": "^0.8.7",
     "service-runner": "^0.3.0",
-    "swagger-router": "^0.2.5",
+    "swagger-router": "git+https://github.com/gwicke/swagger-router.git#template_fixes",
```

Comment:
This can now be replaced with `^0.2.7` since that version contains the code pointed to here.